docs/web-apps/automated-testing/testcafe.md

```diff
@@ -152,3 +152,4 @@ You can run `saucectl` locally via Docker ([Installation Requirements](https://d
 * [Quickstart](/web-apps/automated-testing/testcafe/quickstart): Use our demo repo to quickly set up and run a sample TestCafe project and test to see the results.
 * [Run Your own Tests](/testrunner-toolkit/configuration/testcafe): Customize `saucectl` to run your existing tests just by modifying the `config.yml` file for your project.
 * [Incorporate saucectl in your pipeline](/testrunner-toolkit/integrations): TestCafe on Sauce supports CI integrations with Cirlce CI, GitLab, Jenkins, and GitHub Actions.
+* Use the [TestCafe Sauce Labs Plugin](https://github.com/DevExpress/testcafe-browser-provider-saucelabs) to initiate your tests directly from TestCafe.
```

Comment:
Hm, would be good to **emphasize** that this is an **alternative** to the first `Quickstart`. The first three bullet points are all related to `saucectl`. While this approach differs greatly. If I'm just scanning these "getting started points", I might miss the significance of this one.

Reply:
Thanks, @alexplischke. So do I understand correctly that the plugin allows you to connect to your Sauce Labs account _from_ TestCafe project and configure and run your tests from there, and they will use SL OS/Browser environments and shows up in SL test results?

Reply:
Correct! 💪

Reply:
I have rewritten to hopefully distinguish this as an alternative means of running. @alexplischke -- please kindly re-review, keeping in mind that Meredith is somewhat protective of our use of "Quickstart" -- we can only use that term when the procedure meets the strictly defined guidelines. :)